aether/sdk/tests/webpackfiles/webpack-stats.json

```diff
@@ -0,0 +1 @@
+{"status": "done", "chunks": {"common": [{"name": "common-20191210225738-CDN-static-files.js", "path": "/code/bundles/common-20191210225738-CDN-static-files.js", "pubicPath": "/ui-assets/static/common-20191210225738-CDN-static-files.js"}], "pipeline": [{"name": "pipeline-20191210225738-CDN-static-files.js", "path": "/code/bundles/pipeline-20191210225738-CDN-static-files.js", "pubicPath": "/ui-assets/static/pipeline-20191210225738-CDN-static-files.js"}], "styles": [{"name": "styles-20191210225738-CDN-static-files.css", "path": "/code/bundles/styles-20191210225738-CDN-static-files.css", "pubicPath": "/ui-assets/static/styles-20191210225738-CDN-static-files.css"}, {"name": "styles-20191210225738-CDN-static-files.js", "path": "/code/bundles/styles-20191210225738-CDN-static-files.js", "pubicPath": "/ui-assets/static/styles-20191210225738-CDN-static-files.js"}]}, "startTime": 1576058308786, "endTime": 1576058331660}
```

Comment:
You only need to include one simple entry to test that the file content is changed with he CDN path...